packages/forma-36-react-components/src/components/Notification/NotificationsManager.tsx

```diff
@@ -177,6 +177,7 @@ export function NotificationsManager({
             duration={item.duration}
             key={item.id}
             hasCloseButton={item.canClose}
+            // eslint-disable-next-line react/jsx-handler-names
             onClose={item.close}
```

Comment:
Why not renaming this one?

Reply:
because an item has a property called `.close` and we can not change it

Reply:
Why can't we change it? I thought that was the reason for having the `forma-v4` branch

Reply:
because this is a part of the public API for Notification component

```
const notification = Notification.success();
notification.close();
```

Reply:
I still don't get it 100%, but I guess it's only me so no need for any further explanations ✌️